### ISSUE TITLE
URL constructor error while using proxy host

### ISSUE THREAD

**Issue by KohanTorabi** [DataDog/browser-sdk#529]
When I set the proxy host parameter in datadogLogs:
``` 
datadogLogs.init({
  clientToken: '<DATADOG_CLIENT_TOKEN>',
  site: 'datadoghq.com',
  proxyHost: "dd.*****.com"
});
````
This error appears:
```
Uncaught (in promise) TypeError: Failed to construct 'URL': Invalid URL
    at s (urlPolyfill.ts:38)
    at l (urlPolyfill.ts:20)
    at c (urlPolyfill.ts:16)
    at s (configuration.ts:214)
    at n (errorCollection.ts:163)
    at errorCollection.ts:160
    at fetchProxy.ts:92
    at Array.forEach (<anonymous>)
    at fetchProxy.ts:92
    at tslib.es6.js:100
    at Object.next (tslib.es6.js:81)
    at o (tslib.es6.js:71)
```
As far as I investigated it comes from `buildUrl` function in:
[https://github.com/DataDog/browser-sdk/blob/b3bf73efe65cdc2ebece5bd31dfc80124d66a991/packages/core/src/urlPolyfill.ts#L36](url)

My proxy works well and I can get the logs successfully but this error should be solved

**Comment by BenoitZugmeyer** [comment]
Thank you for the heads up, this issue has been fixed and released in v1.20.0. Let us know if you encounter any other issue!

**Comment by electricsheep** [comment]
Had an issue with v1.19.0 using RUM with a proxy where there were still entries coming through in the explorer but they were missing the data on Performance, Resources etc. Upgrading to v.1.20.0 and presumably this related fix has resolved it.
